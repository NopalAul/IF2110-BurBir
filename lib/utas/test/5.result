lib/utas/test/5.in lib/utas/test/5.out: TRUE
