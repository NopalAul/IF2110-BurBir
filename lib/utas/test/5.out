ucup
tes2
2